gradle.properties

```diff
@@ -1,4 +1,4 @@
 version=23.1.0-SNAPSHOT
 
 org.gradle.welcome=never
-org.gradle.jvmargs=-Xmx1g
+org.gradle.jvmargs=-Xmx3g
```

Comment:
this is in another PR right?

Reply:
it is in another pr at 2g, which still wasn't enough for me. i'll revert it here, and we can increment it in another pr if 2g is really insufficient for most devs.